### PR TITLE
Add Sign up

### PR DESCRIPTION
**Pull request by ace0625**

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -75,8 +89,7 @@ export default class App extends React.Component {
           style={[Styles.textDropdownInnerContainer, (style) ? style : {}]}
           activeOpacity={1}
           onPress={() => {
-            this.setState({currentScene: text});
-            Actions.refresh();
+            Actions.refresh(this.setState({currentScene: text}));
```
Comment (ochanje210) [inline app/app.js:80]: what is the difference from the previous?

File: app/components/signIn/auth.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      },
+
+      onFailure: (transaction) => {
+        reject(transaction.getError().message);
+      }
+    });
+  });
+}
+
+
```
Comment (ochanje210) [inline app/components/signIn/auth.js]: u can delete this extra blank line

Comment (festinalente91) [reply app/components/signIn/auth.js]: updated

File: app/app.js (hunk 1 above)
Comment (ace0625) [reply app/app.js:80]: For just tryna make that shorter.

Comment (ochanje210) [reply app/app.js:80]: ok

**Comment by ochanje210** [comment]
lgtm

[Approved with PullApprove](https://pullapprove.com/recbook/recbook-client/pull-request/34/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=recbook/recbook-client)

**Pull request merged**